mlflow/store/abstract_store.py

```diff
@@ -100,12 +100,18 @@ def rename_experiment(self, experiment_id, new_name):
     @abstractmethod
     def get_run(self, run_uuid):
         """
-        Fetch the run from backend store
-
-        :param run_uuid: Unique identifier for the run
-
-        :return: A single :py:class:`mlflow.entities.Run` object if it exists,
-            otherwise raises an exception
+        Fetch the run from backend store. The resulting :py:class:`Run <mlflow.entities.Run>`
+        contains a collection of run metadata - :py:class:`RunInfo <mlflow.entities.RunInfo>`,
+        as well as a collection of run parameters, tags and metrics -
```

Comment:
Nit: parameters, tags and metrics -> parameters, tags, and metrics

Reply:
Added comma!